modules/gdscript/gdscript_parser.cpp

```diff
@@ -4433,7 +4462,7 @@ void GDScriptParser::TreePrinter::print_variable(VariableNode *p_variable) {
 			if (p_variable->property == VariableNode::PROP_INLINE) {
 				push_line(":");
 				increase_indent();
-				print_suite(p_variable->getter);
+				print_function(p_variable->getter);
```

Comment:
This might be a bit awkward when printed out since it already prints a sort of signature for the getter, then `print_function` will add another signature. It's better to keep printing only the suite:

```suggestion
				print_suite(p_variable->getter->body);
```

Same thing for the setter.